src/utils/num_utils.ts

```diff
@@ -230,3 +230,13 @@ export function constrainToAxis(
       return [...src];
   }
 }
+
+export function getDataRange(data: ArrayLike<number>): [number, number] {
+  let min = Infinity;
+  let max = -Infinity;
+  for (let i = 0; i < data.length; i++) {
+    min = Math.min(min, data[i]);
+    max = Math.max(max, data[i]);
+  }
+  return [min, max];
```

Comment:
Note that this will fail for `NaN` values. Do we ever expect those to be in loaded volume data?

Reply:
another nice way to do this loop is to initialize min and max to data[0] and then iterate from 1 to length.

Reply:
NaNs are pretty unexpected, I think.  Certainly out of scope for this PR.